examples/jsm/exporters/GLTFExporter.js

```diff
@@ -1182,9 +1187,11 @@ class GLTFWriter {
 
 		if ( ! json.textures ) json.textures = [];
 
+		const mimeType = map.userData.mimeType === 'image/webp' ? 'image/png' : map.userData.mimeType;
+
```

Comment:
This is dumber but it's easier to understand what is trying to do:

```js
let mimeType = map.userData.mimeType;

if ( mimeType === 'image/webp' ) mimeType = 'image/png';
```

Reply:
done.